contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,141 @@
+
+Slow velocity is usually a result of [over-engineering](https://www.joelonsoftware.com/2001/04/21/dont-let-architecture-astronauts-scare-you/). "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
+
+### How to fix it
+
+First, sit with the engineering team and [prioritize your technical debt](https://alexeymk.com/2021/04/01/a-toolset-for-tackling-technical-debt.html). Figure out what is *actually* slowing you down (or likely to) and by how much. Once you have a ranked list, empower your team to fix it and assign 10%-20% of engineering time to do so.
+
+Second (and more importantly), urge your team to take shortcuts. The role of a growth engineer is not necessarily to build features, but rather to validate whether a feature should exist. 
+
+With this in mind, a good team exercise is to look at your last 20 experiments and identify which ones you could have validated faster. Thomas Owers, former growth lead at [Let's Do This](https://www.letsdothis.com/), has [good advice](/blog/how-to-start-a-growth-team#thomass-golden-rule-make-your-code-changes-as-small-as-possible) on this:
+
+"Instead of asking 'How long will a project take?', ask 'What can you do in only 1 week? 3 days? Half a day?'. Often, you'll find that you can reduce the scope of the change while still being able to validate your hypothesis".
+
+## Problem 2: Your experiment win rate is low
+
+You keep trying new ideas, but none of them are succeeding. 
+
+"You need better ideas," says Alexey, "and, counter-intuitively, the way to do this is to spend more time adding clear estimates to your current ideas."
+
+Typically, teams prioritize their roadmaps by using frameworks such as [RICE](https://www.intercom.com/blog/rice-simple-prioritization-for-product-managers/) and assigning t-shirt size estimates ("S", "M", "L"). This doesn't go deep enough and leads to teams wasting time by building pointless things. 
+
+Spending only five minutes estimating the impact from something your team will spend weeks on doesn't make sense.
+
+### How to fix it 
+
+Once you have shortlisted the ideas you want to work on, the next step is to **estimate the dollar impact of each one**.  
+
+Then, use this dollar amount together with your effort estimates to calculate a prioritization score:
+
+![How to calculate your EVELYN score](../images/blog/how-to-fix-your-growth-team/evelyn-score.png)
+
+Because you'll have a clearer idea of what to expect from your experiments, the result is a higher win rate.
+
+This approach was made famous by Facebook Messenger's VP of growth Darius Contractor and his [EVELYN framework](https://www.drift.com/blog/secret-to-faster-growth/) (He even shares a [handy sheet with real examples you can copy](https://airtable.com/appzs6HT2ggrJt9pN/shrInOxl0SL1Xmauk/tblTQFEMcXAaEgSpM/viwZessbXpiA8q2jd?blocks=bipMQFWTjqu0HI3wv)).
```

Comment:
I think it is worth noting that the calculation graphic above is the EVELYN framework. Without that, readers don't have any way to know what EVELYN is (without clicking out of the site).

Reply:
EVELYN is slightly deeper than this:

> . The idea is it’s pretty much a experiment tracker, an organizing tool that allows you to put in all your experiments, prioritize them well, track them from specing, designing, building and completion and also see how you did, like opportunity sizing on the way in, look at the results on the way out, and overall give you a set of processes and a storage source of truth for your experiments that I think can really help a team accelerate how they build and build the right experiments, getting you to more growth wins faster.

I dont really want to go to deep into EVELYN, as I think thats a blog post in itself. I think the important bit is the dollar amount tracking.

I might reach out to Darius at some point and see if we can write an EVELYN guide with him